src/renderer/api/kube-object.ts

```diff
@@ -122,12 +122,15 @@ export class KubeObject implements ItemObject {
     return this.metadata.namespace || undefined;
   }
 
-  // todo: refactor with named arguments
+  getTimeDiffFromNow(): number {
+    return new Date().getTime() - new Date(this.metadata.creationTimestamp).getTime();
+  }
+
   getAge(humanize = true, compact = true, fromNow = false) {
     if (fromNow) {
-      return moment(this.metadata.creationTimestamp).fromNow();
+      return moment(this.metadata.creationTimestamp).fromNow(); // "string"
```

Comment:
Why do we need this comment, is `getAge` supposed to return only `number` or `number | string`. Wouldn't it be better to explicitly mark the return type?

Reply:
This is done more to inform why not used `getTimeDiffFromNow(): number` from above.